web/sites/default/settings.php

```diff
@@ -101,6 +101,6 @@
 }
 
 // Silta cluster configuration overrides.
-if (isset($_ENV['SILTA_CLUSTER']) && file_exists($app_root . '/' . $site_path . '/settings.silta.php')) {
+if (null !== getenv('SILTA_CLUSTER') && file_exists($app_root . '/' . $site_path . '/settings.silta.php')) {
```

Comment:
The strict comparison made here will go through as `getenv()` doesn't return null but `false`, so this include would be applied on Lando as well if the file is present.

Reply:
Thanks, @sampoturve!

Reply:
Should be solved in 888be19.